tests/helpers/helpers_test.go

```diff
@@ -50,7 +50,7 @@ const (
 
 var (
 	serverAddress = localhostAddress + ":" + fmt.Sprint(port)
-	testError     = fmt.Errorf("test error")
+	testError     = fmt.Errorf("test error") //nolint:staticcheck // ST1012
```

Comment:
this should be `fmt.Error` or `errors.New`, not `Errorf` as there is no formating

Reply:
`fmt.Error` does not seem to exit

Reply:
then `errors.New`

Reply:
Updated to errors.New